### DISCUSSION TITLE
Add .config folder to build with cmake

### DISCUSSION THREAD

**Discussion by ShadowOfPripyat** [ppy/osu#16297] [Ideas] [answered]
Hello, I have a Raspberry Pi 4 (8 GB) and i would like to compile Osu! For its 64bit Raspbian. I don't find any way to compile; And i have VS Code with all listed requirements.
I would appreciate any help given.

Greetings From Spain!

**Comment by peppy** [comment, marked as answer]
that's not how it works sorry.

**Reply by ShadowOfPripyat** [reply]
Okay. Then, how it works ? What do i need ppy?

**Reply by bdach** [reply]
https://github.com/ppy/osu#building
